RandLAPACK/drivers/rl_cqrrpt.hh

```diff
@@ -42,8 +42,9 @@ class CQRRPT : public CQRRPTalg<T> {
         /// which defaults to 1.
         ///
         /// The algorithm allows for choosing the rank estimation scheme either naively, through looking at the
-        /// diagonal entries of an R-factor from QRCP or via finding the smallest k such that ||A[k:, k:]||_F <= tau_trunk * ||A||_F.
+        /// diagonal entries of an R-factor from QRCP or via finding the smallest k such that ||A[k:, k:]||_F <= tau_trunk * ||A||_x.
         /// This decision is controlled through 'naive_rank_estimate' parameter, which defaults to 1.
+        /// The choice of norm ||A||_x, either 2 or F, is controlled via 'use_fro_norm'.
```

Comment:
I can't comment on Line 58 so I comment here. Please use a more informative variable name than ``t`` in the constructor.

Reply:
Done.